### ISSUE TITLE
Improve kubernetes-client/gen documentation for {CLIENT_LANGUAGE}.sh 

### ISSUE THREAD

**Issue by marie-yau** [kubernetes-client/gen#85]
It's not clear that the PACKAGE_NAME in the settings files for python.sh (or any language).sh file needs to be specified as "client".  Providing an example of the settings file for python-client, python.sh: 

Example settings file for python.sh:

export KUBERNETES_BRANCH="master" 
export CLIENT_VERSION="8.0.0b1"
export PACKAGE_NAME="client"

Note: For generating the client for any language, the PACKAGE_NAME should be "client".
          You can use the latest version for the CLIENT_VERSION. It's displayed here for
          the python-client ( https://github.com/kubernetes-client/python ), and similarly
          for other language clients.

Will generate a PR to check this in.

**Comment by Sai-Adarsh** [comment]
Still open @marie-yau , shall I work?

**Comment by marie-yau** [comment]
Hi, you can work on it.

**Comment by Sai-Adarsh** [comment]
## Settings to add PACKAGE_NAME for {CLIENT_LANGUAGE}.sh
 For generating the client for any language, the PACKAGE_NAME should be "client", along with the latest CLIENT_VERSION.
### Python-Client
* Add these to python.sh
```sh
export KUBERNETES_BRANCH="master"
export CLIENT_VERSION="8.0.0b1"
export PACKAGE_NAME="client"
```

### C Sharp
* Add these to csharp.sh
```sh
export KUBERNETES_BRANCH="master"
export PACKAGE_NAME="client"
```

### Go
* Add these to go.sh
```sh
export KUBERNETES_BRANCH="master"
export CLIENT_VERSION="0.1.0a1"
export PACKAGE_NAME="client"
```

### Haskell
* Add these to haskell.sh
```sh
export KUBERNETES_BRANCH="master"
export PACKAGE_NAME="client"
```

### Java
* Add these to java.sh
```sh
export KUBERNETES_BRANCH="master"
export PACKAGE_NAME="client"
```

### Javascript
* Add these to javascript.sh
```sh
export KUBERNETES_BRANCH="master"
export PACKAGE_NAME="client"
```

### Python Asyncio
* Add these to python-asyncio.sh
```sh
export KUBERNETES_BRANCH="master"
export PACKAGE_NAME="client"
```

### Ruby
* Add these to ruby.sh
```sh
export KUBERNETES_BRANCH="master"
export PACKAGE_NAME="client"
```

### Typescript
* Add these to typescript.sh
```sh
export KUBERNETES_BRANCH="master"
export PACKAGE_NAME="client"
```

**Comment by Sai-Adarsh** [comment]
> ## Settings to add PACKAGE_NAME for {CLIENT_LANGUAGE}.sh
> For generating the client for any language, the PACKAGE_NAME should be "client", along with the latest CLIENT_VERSION.
> 
> ### Python-Client
> * Add these to python.sh
> 
> ```shell
> export KUBERNETES_BRANCH="master"
> export CLIENT_VERSION="8.0.0b1"
> export PACKAGE_NAME="client"
> ```
> 
> ### C Sharp
> * Add these to csharp.sh
> 
> ```shell
> export KUBERNETES_BRANCH="master"
> export PACKAGE_NAME="client"
> ```
> 
> ### Go
> * Add these to go.sh
> 
> ```shell
> export KUBERNETES_BRANCH="master"
> export CLIENT_VERSION="0.1.0a1"
> export PACKAGE_NAME="client"
> ```
> 
> ### Haskell
> * Add these to haskell.sh
> 
> ```shell
> export KUBERNETES_BRANCH="master"
> export PACKAGE_NAME="client"
> ```
> 
> ### Java
> * Add these to java.sh
> 
> ```shell
> export KUBERNETES_BRANCH="master"
> export PACKAGE_NAME="client"
> ```
> 
> ### Javascript
> * Add these to javascript.sh
> 
> ```shell
> export KUBERNETES_BRANCH="master"
> export PACKAGE_NAME="client"
> ```
> 
> ### Python Asyncio
> * Add these to python-asyncio.sh
> 
> ```shell
> export KUBERNETES_BRANCH="master"
> export PACKAGE_NAME="client"
> ```
> 
> ### Ruby
> * Add these to ruby.sh
> 
> ```shell
> export KUBERNETES_BRANCH="master"
> export PACKAGE_NAME="client"
> ```
> 
> ### Typescript
> * Add these to typescript.sh
> 
> ```shell
> export KUBERNETES_BRANCH="master"
> export PACKAGE_NAME="client"
> ```

@marie-yau is this fine. Also CLIENT_VERSION isn't provided in some client-language repos

**Comment by Sai-Adarsh** [comment]
@marie-yau  is this fine? I'll add this to the doc.

**Comment by Sai-Adarsh** [comment]
My branch: https://github.com/Sai-Adarsh/gen

**Comment by Sai-Adarsh** [comment]
@marie-yau could you review the PR?

**Comment by roycaihw** [comment]
it looks like the documentation improvement has been merged in https://github.com/kubernetes-client/gen/pull/86. @marie-yau Is there any change that need to be made?

**Comment by Sai-Adarsh** [comment]
> it looks like the documentation improvement has been merged in #86. @marie-yau Is there any change that need to be made?

It's an example of settings file for python-client. I guess the task is to write similar documentation for all language clients?

**Comment by roycaihw** [comment]
@Sai-Adarsh the pattern is similar for different languages and there seems to be no language-specific setting (as https://github.com/kubernetes-client/gen/issues/85#issuecomment-469769546 shows). I suppose mentioning ["similarly for other language clients"](https://github.com/kubernetes-client/gen/pull/86/files#diff-04c6e90faac2675aa89e2176d2eec7d8R36) is sufficient?
